### DISCUSSION TITLE
Errors

### DISCUSSION THREAD

**Discussion by OdibalAvatar** [kaangiray26/forte#114] [Q&A]
Good evening!

I am getting this error when trying to login:

Uncaught ReferenceError: log_in is not defined
    onclick http://xx.xx.xx.xx:3000/login:1

I initially set up Forte, changed the password, etc.  When I try to login now, the browser drops this on me (firefox or chrome).

**Comment by kaangiray26** [comment]
Hi, I've released a new version that addresses this issue. Version 4.1 should fix this issue.
